lib/gui/css/main.css

```diff
@@ -6487,6 +6487,12 @@ svg-icon {
   max-height: 200px;
   overflow-y: auto; }
 
+.modal-file-selector-modal {
+  min-width: 98vw; }
+  .modal-file-selector-modal > .modal-content {
+    min-height: 95vh;
+    max-height: 95vh; }
```

Comment:
These should also be `calc(100vh - 10px)` like the Wifi modal, and ideally only specify these in one location like the Wifi modal if possible.